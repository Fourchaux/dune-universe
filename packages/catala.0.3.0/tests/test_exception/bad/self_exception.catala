## [Test]

```catala
new scope A:
  param y content int

scope A:
  label base_y
  exception base_y
  def y := 0
```
