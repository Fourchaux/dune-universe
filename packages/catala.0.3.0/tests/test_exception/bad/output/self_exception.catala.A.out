[ERROR] Cannot define rule as an exception to itself
[ERROR] 
[ERROR]  --> test_exception/bad/self_exception.catala
[ERROR]   | 
[ERROR] 9 |   exception base_y
[ERROR]   |             ^^^^^^
[ERROR]   + Test
